### ISSUE TITLE
"Setup and run" might be more effective if it automatically saved

### ISSUE THREAD

**Issue by lukejharmon** [arborworkflows/ArborWebApps#136]
When I am tweaking a workflow, I often change something and then click "setup and run" - and then feel surprised that the change had no effect. The problem is that I have not saved the change, and so the "old" workflow is being run rather than what is on the screen.

To me, "Setup and run" implies "run what is on the screen right now." I think the remedy is to make this button save what is up, then setup and run. You could give an option in a popup if you want "do you want to save first" - or a checkbox or something. But even making this an invisible default behavior would, to me, be fine.

**Comment by jeffbaumes** [comment]
Yes this has nagged me also - it's not obvious which version you will be using. At the least there should be a save whenever you click "setup and run" because I cannot think of a case where you'd want to run the out-of-date version.
